modules/social_features/social_follow_content/config/install/message.template.create_comment_following_node.yml

```diff
@@ -26,7 +26,7 @@ text:
     value: "<p><a href=\"[message:author:url:absolute]\">[message:author:display-name]</a> commented on [social_comment:parent_entity_author]'s [social_comment:commented_content_type] <a href=\"[message:field_message_related_object:entity:url:absolute]\">[message:node-title]</a> you are following</p>\r\n"
     format: full_html
   -
-    value: "<p><a href=\"[message:author:url:absolute]\">[message:author:display-name]</a> commented on [social_comment:parent_entity_author]'s [social_comment:commented_content_type] <a href=\"[message:field_message_related_object:entity:url:absolute]\">[message:node-title]</a> you are following</p>\r\n\r\n<table class=\"comments\">\r\n<tr>\r\n<td class=\"post\"><strong>[social_comment:comment_author]</strong> <small>• [social_comment:comment_date]</small>\r\n<p>[social_comment:comment_text]</p>\r\n</td>\r\n</tr>\r\n</table>\r\n<table class=\"btn-wrapp\">\r\n\t<tbody>\r\n\t\t<tr>\r\n\t\t\t<td class=\"align-center\"><a class=\"btn-link btn-link-bg btn-link-one\" href=\"[social_comment:comment_reply_link_html]\">Reply to this comment </a></td>\r\n\t\t</tr>\r\n\t</tbody>\r\n</table>"
+    value: "<p><a href=\"[message:author:url:absolute]\">[message:author:display-name]</a> commented on [social_comment:parent_entity_author]'s [social_comment:commented_content_type] <a href=\"[message:field_message_related_object:entity:url:absolute]\">[message:node-title]</a> you are following</p>\r\n\r\n<table class=\"comments\">\r\n<tr>\r\n<td class=\"post\"><strong>[social_comment:comment_author]</strong> <small>• [social_comment:comment_date]</small>\r\n<p>[social_comment:comment_text]</p>\r\n</td>\r\n</tr>\r\n</table>\r\n<table class=\"btn-wrapp\">\r\n\t<tbody>\r\n\t\t<tr>\r\n\t\t\t<td class=\"align-center\"><a class=\"btn-link btn-link-bg btn-link-one\" href=\"[social_comment:comment_reply_link]\">Reply to this comment </a></td>\r\n\t\t</tr>\r\n\t</tbody>\r\n</table>"
```

Comment:
Shouldn't `social_comment:comment_reply_link ` be `social_comment:commented_entity_link`? 🤔